pkg/controllers/nodeclass/status/ami.go

```diff
@@ -58,9 +62,15 @@ func (a *AMI) Reconcile(ctx context.Context, nodeClass *v1.EC2NodeClass) (reconc
 		return v1.AMI{
 			Name:         ami.Name,
 			ID:           ami.AmiID,
+			Deprecated:   lo.Ternary(ami.Deprecated, lo.ToPtr(ami.Deprecated), nil),
```

Comment:
Can we also add this new field to our docs as well? https://karpenter.sh/docs/concepts/nodeclasses/#statusamis

Reply:
Updated!